src/core/Akka/Routing/Broadcast.cs

```diff
@@ -185,60 +145,116 @@ public override RouterConfig WithFallback(RouterConfig routerConfig)
         {
             return OverrideUnsetConfig(routerConfig);
         }
-    }
 
-    /// <summary>
-    /// This class represents a <see cref="Group"/> router that sends messages it receives to all of its routees.
-    /// </summary>
-    public class BroadcastGroup : Group
-    {
-        /// <summary>
-        /// This class represents a surrogate of a <see cref="BroadcastGroup"/> router.
-        /// Its main use is to help during the serialization process.
-        /// </summary>
-        public class BroadcastGroupSurrogate : ISurrogate
+        private RouterConfig OverrideUnsetConfig(RouterConfig other)
```

Comment:
I think you resolved this issue by making sure the underlying fluent methods on RouterConfig all return the correct implementation class now, whereas this did some of that before. Plus we're using null instead of NoRouter - so I think this is good.